brave/brave-core
# WIP: IPFS extension

Fixes brave/brave-browser#1042

PR based on work done on Tor like: #316 

## Submitter Checklist:


## Test Plan:
- Extension download and 
  - installation
  - Start with a clean profile
  - Launch Brave
  - Visit chrome://components
  - Verify that Ipfs Client Updater is listed and version number is not 0.0.0.0
  - Verify that name of component contains platform identifier (Windows/Mac/Linux)
  - Command-line switch to disable extension

- Start with a clean profile
  - Launch Brave with `--disable-ipfs-client-updater-extension`
  - Visit chrome://components and verify that Ipfs Client Updater is not listed

## Reviewer Checklist:
